tests/core20/remove/task.yaml

```diff
@@ -0,0 +1,30 @@
+summary: Check that removal of essential snaps does not work
+
+execute: |
+    echo "Ensure snapd cannot be removed"
+    if snap remove snapd; then
+        echo "The snapd snap should not be removable"
+        exit 1
+    fi
+    echo "Ensure core20 cannot be removed"
+    if snap remove core20; then
+        echo "The core20 snap should not removable"
+        exit 1
+    fi
+
+    echo "Install a snap that requires core as the base"
+    snap install test-snapd-sh
+    snap list | MATCH '^core '
+    if snap remove core; then
+        echo "core should not be removalble because test-snapd-tools needs it"
+        exit 1
+    fi
+
+    echo "But core can be removed again once nothing on the system needs it"
+    snap remove test-snapd-sh
+    snap remove core
+
+    if snap list | MATCH '^core '; then
```

Comment:
same as above, can this be `snap list core` ?

Reply:
done, thanks